react-backend/routes/cbauthentication/cb_login.js

```diff
@@ -20,10 +19,14 @@ router.post('/', (req, res, next) => {
 
   cbLogin(formEmail, passwordHash)
     .then(exists => {
-      const token = jwt.sign({ email: formEmail }, process.env.SECRET);
-      const loggedIn = exists ? { success: true, token } : { success: false };
-
-      res.send(loggedIn);
+      if (exists) {
+        const token = jwt.sign({ email: formEmail }, process.env.SECRET);
+        const loggedInResponse = { success: true, token };
+        res.send(loggedInResponse);
+      } else {
+        const errorResponse = { success: false };
+        res.status(401).send(errorResponse);
```

Comment:
As far as I can see, no part of the client code explicitly relies on this route returning a `200` code for an unsuccessful response, but could you confirm you've double checked manually (e.g. by clicking through and trying yourself?)

Reply:
Totes did, I thought I had done the bad thing, but turns out it was already broked. See #207

Reply:
Cool, thanks, just the logs then